### ISSUE TITLE
Show @deprecated text in the output

### ISSUE THREAD

**Issue by ghost** [jsdoc2md/jsdoc-to-markdown#90]
Hi,

I use the v1.3.7

In the source, I have added :

```
    @deprecated Use {@link Di#set} instead
    @method
```

And the output is :

```
Deprecated
Kind: instance method of Di
```

**The text "Use Di#set instead" doesn't exist in output. Original jsdoc template shows it.**

Thanks

**Comment by 75lb** [comment]
Hi, currently text set with the `@deprecated` tag is not shown in the output.. i will add this.

**Comment by myarmolinsky** [comment]
Hi, any updates regarding this issue? I'm on ^9.0.0 and text following `@deprecated` is still not included in the output

**Comment by 75lb** [comment]
Hi, I have a couple of generated `@deprecated` tags [here](https://github.com/jsdoc2md/testbed/blob/4c26194f5dd9ae311ee9c8daaac1d80799f66baa/build/jsdoc/global/deprecated/3-dmd.md) in my testbed.. Where do you think the deprecated text should appear? 

At the end of the line in italics like the below? 

---

~~deprecatedFunction2()~~
***Deprecated since version 2.0***

**Kind**: global function

**Comment by myarmolinsky** [comment]
Hi, I expected it to appear right next to the ***Deprecated***, here's my commit in my repo from today: https://github.com/balena-io/balena-sdk/pull/1504/commits/f87ee9f158fae22269be2a8426ecaaae600190ea

You can see the ***Deprecated*** at the bottom of the DOCUMENTATION.md diff, and the warning below it does not come from the note I left by the `@deprecated` comment. Instead, it comes from me manually adding it to the description as a temporary fix for it not appearing in the documentation after running `npm run build`.

Perhaps we're missing something with the way we use `jsdoc2md` https://github.com/balena-io/balena-sdk/blob/f98ec86d8de4941c67095887c4d7ec99e929ddcf/package.json#L52

I'll have a look again tomorrow, but if you have any idea what may be causing the note to not appear on our end I'm happy to hear your thoughts

**Comment by 75lb** [comment]
Fixed and released in [v9.1.0](https://github.com/jsdoc2md/jsdoc-to-markdown/releases/tag/v9.1.0) - let me know if there's anything else.

**Comment by 75lb** [comment]
[Amended.. ](https://github.com/jsdoc2md/jsdoc-to-markdown/releases/tag/v9.1.1)
